.eslintrc.js

```diff
@@ -4,9 +4,7 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
-    'max-len': 0,
```

Comment:
It's a good practice to set a value for 'max-len' rule to enforce a maximum line length. For example: 'max-len': ['error', 80]